experimental/maven-plugin/src/main/java/io/github/slsa-framework/SlsaVerificationMojo.java

```diff
@@ -88,7 +89,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                         element(name("groupId"), artifact.getGroupId()),
                         element(name("artifactId"), artifact.getArtifactId()),
                         element(name("version"), artifact.getVersion()),
-                        element(name("type"), "intoto.build.slsa"),
+                        element(name("type"), "build.slsa"),
```

Comment:
@AdamKorcz please confirm this change is correct

Reply:
It is correct.